utils.go

```diff
@@ -107,10 +108,9 @@ func readBool(input string) (value bool, valid bool) {
 ******************************************************************************/
 
 func parseDateTime(b []byte, loc *time.Location) (time.Time, error) {
-	const base = "0000-00-00 00:00:00.000000"
 	switch len(b) {
 	case 10, 19, 21, 22, 23, 24, 25, 26: // up to "YYYY-MM-DD HH:MM:SS.MMMMMM"
-		if string(b) == base[:len(b)] {
+		if bytes.Equal(b, zeroDateTime[:len(b)]) {
```

Comment:
Why?
Do you expect performance improvement? or readability improvement?

Reply:
Both actually. No need for a constant variable `base` here.